analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt

```diff
@@ -133,11 +132,16 @@ abstract class AbstractIntegrationSpec : StringSpec() {
                     VersionControlSystem.forType(result.project.vcsProcessed.type) shouldBe
                             VersionControlSystem.forType(pkg.vcs.type)
                     result.project.vcsProcessed.url shouldBe pkg.vcs.url
-                    result.project.scopes shouldNot beEmpty()
-                    result.packages shouldNot beEmpty()
+                    result.project.hasScopes() shouldBe true
+                    (results.dependencyGraph != null || result.packages.isNotEmpty()) shouldBe true
```

Comment:
Can we split this into two assertions wrapped in `forOne` instead? And could we use a similar appraoch to get rid of the `Project.hasScopes()` helper function?

Reply:
For details see https://kotest.io/docs/assertions/inspectors.html.

Reply:
From the description of inspectors, it is not clear to me how `forOne` could be used in this scenario. Do you mean something like `listOf(results.dependencyGraph != null, result.packages.isNotEmpty()).forOne { 
  it shouldBe true 
}` ?
I seem to miss something obvious.

Reply:
> Do you mean something like `listOf(results.dependencyGraph != null, result.packages.isNotEmpty()).forOne { it shouldBe true }` ?

Yes.

Reply:
Changed for both cases.